pkg/node/node_connectivity_checker.go

```diff
@@ -356,7 +363,7 @@ func (s *Service) populateTargetsInCache(array *array.PowerStoreArray) {
 				}
 				break
 			}
-			log.Debugf("Portal %s is not rechable from the node", address.Portal)
+			log.Debugf("Portal %s is not rechable from the node", ipAddress)
```

Comment:
_ipAddress_ will not have any value populated at this line.

Reply:
Taken care